documentation/source/users/rmg/input.rst

```diff
@@ -944,3 +941,41 @@ For example ::
 		maxNumSpecies=100
 	)
 
+Restarting from a Seed Mechanism
+=================================
+The only method for restarting an RMG-Py job is to restart the job from a seed mechanism. There are
+many scenarios when the user might want to do this, including continuing on a job that ran out of time or crashed as the
+result of a now fixed bug. To restart from a seed mechanism, the block below must be added on to the input file.  ::
+
+    restartFromSeed(path='seed')
+
+The ``path`` flag is the path to the seed mechanism folder, which contains three subfolders that must be titled as
+follows: filters, seed, seed_edge. The path can be a relative path from the input.py file or an absolute path on disk
+
+Alternatively, you may also specify the paths to each of the following files/directories: coreSeed (path to seed
+mechanism folder containing files named ``reactions.py`` and ```dictionary.txt``` that will go into the model core),
+edgeSeed (path to edge seed mechanism folder containing files named ``reactions.py`` and ```dictionary.txt``` that will
+go into the model edge), filters (an h5 binary file storing the uni-, bi-, and optionally tri-molecular thresholds
+generated by RMG from previous run (must be RMG version > 2.4.1)), speciesMap (a YAML file specifying what species are
+at each index in the filters). ::
+
+    restartFromSeed(coreSeed = 'seed/seed'  # Path to core seed folder. Must contain `reactions.py` and `dictionary.txt`
+                   edgeSeed = 'seed/seed_edge'  # Path to edge seed folder containing `reactions.py` and `dictionary.txt`
+                   filters='seed/filters/filters.h5',
+                   speciesMap='seed/filters/species_map.yml')
```

Comment:
Spacing around `=` and indent an extra space

Reply:
Thanks!